src/Model/PageInterface.php

```diff
@@ -113,13 +113,17 @@ public function setChildren(Collection $children): void;
 
     public function addChild(self $child): void;
 
+    public function removeChild(self $child): void;
```

Comment:
I think this is a BC break no? @VincentLanglet

Reply:
Indeed.

I don't see you using those function @Hanmac ; why do you need them ?

Reply:
The BC solution is to add them to Page, but comment them with a NEXT_MAJOR "uncomment", in the interface.
And the place you need them you check if "method_exists"

Reply:
> Indeed.
> 
> I don't see you using those function @Hanmac ; why do you need them ?

Serializer reacts very funky when the Collection you want to load into has addFunction but not a removeFunction

Reply:
> The BC solution is to add them to Page, but comment them with a NEXT_MAJOR "uncomment", in the interface. And the place you need them you check if "method_exists"

phpstan probably needs an ignore for the methods exist check?

Reply:
Yes, we generally add 
```
// @phpstan-ignore-next-line
```